console/console-init/ui/src/modules/connection/utils/utils.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2020, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+import { IConnection } from "modules/connection";
+
+const getFilteredConnectionNames = (connections: IConnection[]) => {
+  return connections && connections.map(connection => connection.name);
```

Comment:
should this be using the connection's hostname rather than its name (the connection name is just uuid on the server btw, the mock's behaviour is wrong)
```suggestion
  return connections && connections.map(connection => connection.hostname);
```